src/analytics/build.sbt

```diff
@@ -7,30 +7,21 @@ dependencyOverrides += "com.fasterxml.jackson.core" % "jackson-databind" % "2.6.
 dependencyOverrides += "com.fasterxml.jackson.module" % "jackson-module-scala_2.11" % "2.6.7"
 
 libraryDependencies ++= Seq(
-  // Deal with GeoMesa dependency that breaks Spark 2.2
-  "org.json4s" %% "json4s-native" % "3.2.11",
-  "org.json4s" %% "json4s-core" % "3.2.11",
-  "org.json4s" %% "json4s-ast" % "3.2.11",
-
-  "org.postgresql" % "postgresql" % "42.2.2",
-
+  postgresql,
   decline,
   sparkHive % Provided,
   sparkJts,
-  gtGeotools exclude("com.google.protobuf", "protobuf-java"),
-  gtS3 exclude("com.google.protobuf", "protobuf-java") exclude("com.amazonaws", "aws-java-sdk-s3"),
-  gtSpark exclude("com.google.protobuf", "protobuf-java"),
-  gtVector exclude("com.google.protobuf", "protobuf-java"),
-  gtVectorTile exclude("com.google.protobuf", "protobuf-java"),
-  "com.google.protobuf" % "protobuf-java" % "2.5.0",
-  vectorpipe exclude("com.google.protobuf", "protobuf-java"),
+  gtGeotools,
+  gtS3,
+  gtSpark,
+  gtVector,
+  gtVectorTile,
+  vectorpipe,
```

Comment:
Care to comment on this change?  I'm not sure why it was there in the first place, and if you changed this, I presume you might know why it was there and why it doesn't need to be.

Reply:
Hopefully...

I ran back through the git history, and this exclusion was added when we integrated GeoMesa into the project: https://github.com/azavea/osmesa/commit/90f5a9bbd2cd22ebbd7f6bf29d7d7670fcc2e43d#diff-7a1b04fa74bea5babe2e064486dc2a76R7-R8

Since then, the exclusion survived the migration from `ingest` -> `analytics` project and a GeoMesa major version upgrade. In the linked commit above we were using geomesa 1.3, now were on 2.3.

So...I don't know why it was originally there. I didn't dig back through geomesa changelogs to see if it was something they required at the time or something we had to add to resolve dependency hell. I removed it and I was still able to build an assembly jar and run a few of the streaming apps without any issues. If you're aware of any other tests or checks I should do to validate this change, I'm happy to run them.